rust_gmf_bench/.cargo/config

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = [ "-C", "target-cpu=native" ]
```

Comment:
This flag makes the binary less portable. Other implementations do not use this and it would be fair not to use this here as well.